code/modules/awaymissions/corpse.dm

```diff
@@ -20,6 +20,10 @@
 	var/corpseidicon = null //For setting it to be a gold, silver, centcomm etc ID
 	var/mutantrace = "human"
 
+/obj/effect/landmark/corpse/New()

```

Comment:
Inherit the New, because it helps with some Byond feature I can't recall.
I think the GC.